ambuda/views/proofing/page.py

```diff
@@ -110,43 +111,53 @@ def add_revision(
     return new_version
 
 
+def _split_graphemes(s: str) -> str:
```

Comment:
I think our diffing logic is now substantial enough that we could move it to its own module, maybe `ambuda.utils.diffs`.